### PR TITLE
Update Terraform Install Page

### PR DESCRIPTION
**Pull request by laurapacilio**
This PR updates:
-  The About Terraform sidecar on the Install page. It's in response to an internal Asana task required for the upcoming launch. I used the overview sentence already have from "What is Terraform?" And then I chose (what I think are) the most likely next places users will want to go as they are installing the product. 
- The featured tutorial slugs. I used all of the Get Started tutorials, as I think they're the most relevant at this stage.
- Adds sidebar menu items. I tried to follow the lead from Vault and Waypoint, which have links to intro pages and quickstart guides.

### CODE REVIEW

File: src/content/terraform/install-landing.json [hunk 1]
```diff
@@ -18,27 +18,27 @@
...
 		"title": "About Terraform",
-		"subtitle": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur eros diam, fringilla ac malesuada vel, faucibus quis mauris.",
+		"subtitle": "Terraform is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share.",
 		"featuredDocsLinks": [
 			{
-				"href": "/terraform/internals",
-				"text": "Internals"
+				"href": "/terraform/intro",
+				"text": "Introduction to Terraform"
 			},
```
Comment (laurapacilio) [inline src/content/terraform/install-landing.json:26]:
```suggestion
			},
			{
			        "href": "/terraform/tutorials#get-started",
				"text": "Configuration Language"
			},
```

File: src/content/terraform/install-landing.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
 {
-	"featuredTutorialsSlugs": [],
+	"featuredTutorialsSlugs": [
+            "terraform/tutorials/aws-get-started",
+	    "terraform/tutorials/azure-get-started",
+	    "terraform/tutorials/docker-get-started",
+	    "terraform/tutorials/gcp-get-started",
+	    "terraform/tutorials/oci-get-started",
+	    "terraform/tutorials/cloud-get-started"
```
Comment (BRKalow) [inline src/content/terraform/install-landing.json:8]: @laurapacilio Are these tutorials or collections? Only tutorials are supported here at the moment

Comment (laurapacilio) [inline src/content/terraform/install-landing.json:8]:
```suggestion
            "/terraform/tutorials/aws-get-started/install-cli",
	    "/terraform/tutorials/azure-get-started/install-cli",
	    "/terraform/tutorials/docker-get-started/install-cli",
	    "/terraform/tutorials/gcp-get-started/install-cli",
	    "/terraform/tutorials/oci-get-started/install-cli",
	    "/terraform/tutorials/cloud-get-started/cloud-login"
```

Comment (laurapacilio) [reply src/content/terraform/install-landing.json]: fixed!! thank you!

File: src/content/terraform/install-landing.json [hunk 3]
```diff
@@ -1,5 +1,12 @@
 {
-	"featuredTutorialsSlugs": [],
+	"featuredTutorialsSlugs": [
+            "/terraform/tutorials/aws-get-started/install-cli",
+	    "/terraform/tutorials/azure-get-started/install-cli",
+	    "/terraform/tutorials/docker-get-started/install-cli",
+	    "/terraform/tutorials/gcp-get-started/install-cli",
+	    "/terraform/tutorials/oci-get-started/install-cli",
+	    "/terraform/tutorials/cloud-get-started/cloud-login"
```
Comment (laurapacilio) [inline src/content/terraform/install-landing.json:8]:
```suggestion
	"featuredTutorialsSlugs": [
            "terraform/tutorials/aws-get-started/install-cli",
	    "terraform/tutorials/azure-get-started/install-cli",
	    "terraform/tutorials/docker-get-started/install-cli",
	    "terraform/tutorials/gcp-get-started/install-cli",
	    "terraform/tutorials/oci-get-started/install-cli",
	    "terraform/tutorials/cloud-get-started/cloud-login"
```

**Review by judithpatudith — APPROVED** [review]

**Comment by judithpatudith** [comment]
This looks good! The only thing I noticed is that in the right sidebar, the **Learn more** link goes to terraform.io. Is that intentional?

File: src/content/terraform/install-landing.json [hunk 4]
```diff
@@ -15,37 +15,41 @@
...
+				"text": "Introduction to Terraform"
 			},
 			{
 				"href": "/terraform/language",
-				"text": "Configuration"
+				"text": "Configuration Language"
 			},
 			{
 				"href": "/terraform/cli/commands",
 				"text": "CLI"
```
Comment (laurapacilio) [inline src/content/terraform/install-landing.json:44]:
```suggestion
				"href": "/terraform/cli/commands",
				"text": "Terraform CLI"
```

**Comment by laurapacilio** [comment]
@ashleemboyer - is there a way to make this say "Featured Tutorials" --> I think it still makes sense in context and it matches @judithpatudith guidance about avoiding the term "Collections" where possible.

Screen Shot 2022-09-06 at 3 08 36 PM

**Comment by laurapacilio** [comment]
@BRKalow why is the next.js check failing? should I be concerned about that 😬

**Comment by BRKalow** [comment]
@laurapacilio I think in this case it's because this PR is coming from your fork of `dev-portal`. Should be okay to ignore!

**Comment by laurapacilio** [comment]
@BRKalow I don't think it will let me merge this without all the checks passing. Which...I think is a nice safety feature, it's just blocking me on this. Please help!

**Comment by BRKalow** [comment]
@laurapacilio merged!

**Pull request merged**